lib/parsers/moov_parser.rb

```diff
@@ -115,5 +118,33 @@ def matches_moov_definition?(io)
     maybe_atom_size >= minimum_ftyp_atom_size && maybe_ftyp_atom_signature == 'ftyp'
   end
 
+  def parse_sample_atom(decoder, atom_tree)
+    video_trak_atom = decoder.find_video_trak_atom(atom_tree)
+
+    stts = begin
```

Comment:
What does `stts` stands for? Also, the `begin` block is unnecessary, you can just do `stts = if video_trak_atom ...`
Same for the code bellow.

Reply:
"stts" is the name of an atom in that format, will add a comment to clarify that